main.cpp

```diff
@@ -51,85 +51,95 @@ arm.flex();                             // 3)
 arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
-//  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  10)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
+//  1)
+//  Noun: person
+//  action 1: the person sleeps
+person.sleep();
+//  action 2: the person eats
+person.eat();
+//  action 3: the person calls their mother
+person.callMom();
+
+//  2) 
+//  Noun: automatic door
+//  action 1: the automatic door waits until it detects a person
+automaticDoor.detectPerson();
+//  action 2: the automatic door opens
+automaticDoor.open();
+//  action 3: the automatic door closes
+automaticDoor.close();
+
+//  3)
+//  Noun: cat
+//  action 1: the cat meows
+cat.meow();
+//  action 2: the cat hisses
+cat.hiss();
+//  action 3: grooms itself
+cat.groomSelf();
+
+//  4)
+//  Noun: bread maker
+//  action 1: the bread maker waits for ingredients
+breadMaker.waitForIngredients();
+//  action 2: mixes ingredients into dough
+breadMaker.mixIngredients();
+//  action 3: cooks dough into bread
+breakMaker.cookDough();
+
+//  5)
+//  Noun: star
+//  action 1: a star emits light
+star.emitLight();
+//  action 2: a star expands
+star.expand();
+//  action 3: a star collapses
+earch.collapse();
+
+//  6) 
+//  Noun: cell
+//  action 1: a cell divides
+cell.divide();
+//  action 2: a cell consumes energy
+cell.consumeEnergy();
+//  action 3: a cell dies
+cell.die();
+
+//  7)
+//  Noun: tree
+//  action 1: a tree grows leaves
+tree.growLeaf();
+//  action 2: a tree sheds leaves
+tree.shedLeaf();
+//  action 3: a tree grows fruit
+tree.growFruit();
+
+//  8)
+//  Noun: answering machine
+//  action 1: an answering machine waits for calls
+answeringMachine.waitForCall();
+//  action 2: an answering machine records messages
+answeringMachine.recordMessage();
+//  action 3: an answering machine plays back recorded messages
+answeringMachine.playRecordedMessage();
+
+//  9)
+//  Noun: snow
+//  action 1: snow falls
+snopw.fall();
+//  action 2: snow accumulates
+snow.accumulate();
+//  action 3: snow melts
+snow.melt();
```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.